<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Invitation</class>
 <widget class="QWidget" name="Invitation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QPushButton" name="pushButton_Home2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>915</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_Ajouter">
    <widget class="QLabel" name="bck_p1_invitation">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>911</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/business-people-working-illustration_23-2148474053.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>781</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(43,70,139,0.1);
border-radius:60px;
background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>0</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/img/plus-circle-solid (1).svg);
border: 3px solid #fa9746;
border-radius:50%;
padding:10px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:2px solid #fa9746;
border-radius:25px;
background-color:transparent;
</string>
      </property>
      <property name="text">
       <string>Inviter</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>136</y>
        <width>741</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font-size:18px;
font-family:Arial;
border:none;

color: black;
margin:0 10px;
background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Personnes</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_personnes_ajouter">
         <property name="styleSheet">
          <string notr="true">font-size:18px;
font-family:Arial;
border:1px solid  #fa9746;
border- radius:3px;
padding: 1px 18px 1px 3px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font-size:18px;
font-family:Arial;
border:none;

color: black;
margin:0 10px;
background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Tables</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_tables_ajouter">
         <property name="styleSheet">
          <string notr="true">font-size:18px;
font-family:Arial;
border:1px solid  #fa9746;
border- radius:3px;
padding: 1px 18px 1px 3px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>211</width>
        <height>72</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QRadioButton" name="radioButton">
         <property name="styleSheet">
          <string notr="true">background-color:transparent;
color:white;</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font-size:18px;
font-family:Arial;
border:none;

color: black;
margin:0 10px;
background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Position</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="radioButton_2">
         <property name="styleSheet">
          <string notr="true">background-color:transparent;
color:white;</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="radioButton_3">
         <property name="styleSheet">
          <string notr="true">background-color:transparent;
color:white;</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="radioButton_4">
         <property name="styleSheet">
          <string notr="true">background-color:transparent;
color:white;</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_Afficher">
    <widget class="QLabel" name="bck_p2_invitation">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>911</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:25px</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>401</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_Modifier">
    <widget class="QLabel" name="bck_p3_invitation">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>921</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/young-people-illustrationjpg.jpg);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>781</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(43,70,139,0.1);
border-radius:60px;
background-color:transparent;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
image: url(:/img/img/edit-solid (1).svg);
border: 3px solid #fa9746;
border-radius:50%;
padding:10px;
background-color:transparent;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>300</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:2px solid #fa9746;
border-radius:25px;
background-color:transparent;
</string>
      </property>
      <property name="text">
       <string>Enregistrer</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>741</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font-size:18px;
font-family:Arial;
border:none;

color: grey;
margin:0 10px;
background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Personnes</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_3">
         <property name="styleSheet">
          <string notr="true">font-size:18px;
font-family:Arial;
border:1px solid  #fa9746;
border- radius:3px;
padding: 1px 18px 1px 3px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font-size:18px;
font-family:Arial;
border:none;

color: grey;
margin:0 10px;
background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Tables</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_4">
         <property name="styleSheet">
          <string notr="true">font-size:18px;
font-family:Arial;
border:1px solid  #fa9746;
border- radius:3px;
padding: 1px 18px 1px 3px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QRadioButton" name="radioButton_7">
     <property name="geometry">
      <rect>
       <x>489</x>
       <y>267</y>
       <width>29</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:18px;</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_6">
     <property name="geometry">
      <rect>
       <x>489</x>
       <y>244</y>
       <width>29</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:18px;</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>489</x>
       <y>221</y>
       <width>29</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:18px;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>260</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:18px;
font-family:Arial;
border:none;

color: grey;
margin:0 10px;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_8">
     <property name="geometry">
      <rect>
       <x>489</x>
       <y>290</y>
       <width>29</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:18px;</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_supprimer">
    <widget class="QLabel" name="bck_p4_invitation">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>911</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/dilemmejpg.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>461</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(43,70,139,0.1);
border-radius:60px;
background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:2px solid #c25711;
border-radius:25px;
color:grey;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
image: url(:/img/img/trash-solid.svg);
border: 3px solid #c25711;
border-radius:50%;
padding:10px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>401</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">font-size:18px;
font-family:Arial;
border:none;

color: grey;
margin:0 10px;
background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Personne</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_5">
         <property name="styleSheet">
          <string notr="true">font-size:18px;
font-family:Arial;
border:1px solid  #fa9746;
border- radius:3px;
padding: 1px 18px 1px 3px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>7</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_Ajouter_Invitation">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Afficher_Invitation">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Modifier_Invitation">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Supprimer_Invitation">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
